web/src/pages/apd/new/ApdNewHITECHFields.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { ChoiceList } from '@cmsgov/design-system';
+import { Controller, useFormContext } from 'react-hook-form';
+
+const ApdNewHITECHFields = ({ setUpdateStatus, updateStatus }) => {
```

Comment:
I think it would be clearer if you called HitechUpdateStatus or UpdateStateHitech